rpms/Dockerfile

```diff
@@ -2,17 +2,24 @@ ARG BASE_IMAGE
 
 FROM ${BASE_IMAGE}
 
-RUN yum install -y epel-release git make yum-utils && \
-    yum-config-manager --enable powertools && \
-    yum install -y rpm-build && \
+RUN yum install -y epel-release git make yum-utils rpm-build && \
     yum group install -y "Development Tools" && \
     yum clean all
 
-WORKDIR /build
-COPY after-checkout-* /build/
-
 ARG BASE_IMAGE
 ENV BASE_IMAGE=${BASE_IMAGE}
 
+RUN if [ "$BASE_IMAGE" = "rockylinux:9" ]; then \
+        dnf install -y dnf-plugins-core && \
+        dnf config-manager --set-enabled crb && \
+        yum clean all; \
+    else \
+        yum-config-manager --enable powertools && \
+        yum clean all; \
+    fi
```

Comment:
I am not sure it's really necessary if we are not installing anything else later, can you try to just remove this entire block or provide a link to a previous build which explain this requirement?

Reply:
https://github.com/Alfresco/imagemagick-build/actions/runs/6312526842/job/17138757252